editions/tw5.com/tiddlers/docker/Using TiddlyWiki with Docker.tid

```diff
@@ -0,0 +1,39 @@
+created: 20230403211230516
+modified: 20230403211230516
+tags: [[TiddlyWiki with Docker]]
+title: Using TiddlyWiki with Docker
+type: text/vnd.tiddlywiki
+
+~TiddlyWiki5 can also be run using Docker. See the [[Dockerfile]] for details.
+
+The image can be built using the following command in the root of the TiddlyWiki git repo:
+```
+docker build --tag tiddlywiki .
+```
+
+It can be run like so:
+```
+# Create a data directory
+mkdir ~/tiddlywiki
```

Comment:
Perhaps it would be clearer for the directory name to be something like `tiddlywiki-data`?

Reply:
Agreed, I'll change that!